### ISSUE TITLE
Accessibility: Add the Ability to set the Warning Colour

### ISSUE THREAD

**Issue by Windchild292** [MegaMek/megamek#1896]
The current warning colour is red, which cannot be easily seen for people with certain forms of colourblindness. #1859 implemented a value in GUIPreferences for the Warning Colour, but there is currently no way to set it outside of editing the saved file. This story is to add a way to implement it in the GUI.
